### PR TITLE
Second pull request-code review-portfoloi(template5)

### PR DESCRIPTION
**Pull request by barath-7**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,17 +18,182 @@
...
+                        <span>C++</span>
+                    </div>
+                    <div class="small-rectangle">
+                        <span>Ruby</span>
+                    </div>
+                </div>
+                <button class="project-visit" type="submit"><span class="button-text">See This Project<img class="arrow-image" src=".img/outline_arrow_forward_white_24dp.png" alt="arr"></span></button>
+            </div>
+        </div>
+        <div class="about-my-section">
```
Comment (vichuge) [inline index.html:156]: - [ ] Try to use a `<section>` tag in place of a div, this is more according to HTML5 and will help SEO purposes. 🐔

File: index.html [hunk 2]
```diff
@@ -18,17 +18,182 @@
...
+            <h2 class="language-heading">Skills</h2>
+            <ul class="language-list">
+                <li>Codekil</li>
+                <li>Codepen</li>
+                <li>Github</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+            </ul>
+        </div>
+    </div>
```
Comment (vichuge) [inline index.html:197]: - [ ] It looks like you forgot to include the `<footer>` section. Please add it. 🍰

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

- [ ] According to the Figma template, this left white border isn't necessary, to remove it could help you to have a better design on your project, please fix this. 🐶
image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (barath-7) [reply index.html:197]: Hi @vichuge, As per the requirement there is no content for the footer. Do you want me to add an empty footer. can you help clarify this

File: style.css [hunk 3]
```diff
@@ -124,3 +125,214 @@ body {
...
+.projects-grid-container {
+  width: 70%;
+  margin: 0 auto;
+  margin-top: 15%;
+}
+
+.projects {
+  display: grid;
+  gap: 10px;
+}
```
[the remark this answers is not in the document]
Comment (barath-7) [reply style.css:148]: Is it mandatory to use flexbox here?

**Comment by barath-7** [comment]
Fixed based on comments

File: index.html [hunk 4]
```diff
@@ -18,17 +18,182 @@
...
+                    <div class="small-rectangle">
+                        <span>C++</span>
+                    </div>
+                    <div class="small-rectangle">
+                        <span>Ruby</span>
+                    </div>
+                </div>
+                <button class="project-visit" type="submit"><span class="button-text">See This Project<img class="arrow-image" src=".img/outline_arrow_forward_white_24dp.png" alt="arr"></span></button>
+            </div>
+        </div>
```
[the remark this answers is not in the document]
Comment (od-c0d3r) [reply index.html]: Great work implement the changes required by the previous reviewer 👍🏼 💯

File: style.css (hunk 3 above)
[the remark this answers is not in the document]
Comment (od-c0d3r) [reply style.css]: Good Work, useing both the Grid and Flexbox in through out your project. 💯

File: index.html [hunk 5]
```diff
@@ -18,17 +18,182 @@
...
+                    <div class="small-rectangle">
+                        <span>Java</span>
+                    </div>
+                    <div class="small-rectangle">
+                        <span>C++</span>
+                    </div>
+                    <div class="small-rectangle">
+                        <span>Ruby</span>
+                    </div>
+                </div>
```
[the remark this answers is not in the document]
Comment (od-c0d3r) [reply index.html]: Great work implement the changes required by the previous reviewer 👍🏼 💯

File: index.html (hunk 1 above)
Comment (od-c0d3r) [reply index.html]: Great work implement the changes required by the previous reviewer 👍🏼 💯

File: index.html (hunk 2 above)
Comment (od-c0d3r) [reply index.html]: All good @barath-7 💯

**Review by od-c0d3r — APPROVED** [review]
Hi @barath-7 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**